src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -288,39 +289,32 @@ def test_omp_single_node_str():
         create(invoke_info)
     schedule = psy.invokes.invoke_list[0].schedule
 
-    _, _ = single.apply(schedule.children[0])
+    single.apply(schedule.children[0])
     omp_single_loop = schedule.children[0]
     out = OMPSingleDirective.node_str(omp_single_loop)
     directive = colored("Directive", Directive._colour)
     expected_output = directive + "[OMP single]"
     assert expected_output in out
 
+
 def test_omp_single_validate_global_constraints():
-    ''' Test the validate_global_constraints method of the OMPSingle directive '''
+    ''' Test the validate_global_constraints method of the OMPSingle
+        directive '''
     _, invoke_info = parse(os.path.join(BASE_PATH, "1_single_invoke.f90"),
                            api="dynamo0.3")
     single = OMPSingleTrans()
     psy = PSyFactory("dynamo0.3", distributed_memory=False).\
         create(invoke_info)
     schedule = psy.invokes.invoke_list[0].schedule
 
-    _, _ = single.apply(schedule.children[0])
+    single.apply(schedule.children[0])
     omp_single_loop = schedule.children[0]
     with pytest.raises(GenerationError) as excinfo:
         schedule.children[0].validate_global_constraints()
-    assert ("OMPSingleDirective must be inside an OMP parallel region but "
-           "could not find an ancestor OMPParallelDirective node" ) in str(excinfo.value)
+    assert ("OMPSingleDirective must be inside an OMP parallel region but " +
+            "could not find an ancestor OMPParallelDirective node") in \
+    str(excinfo.value)
 
-    schedule = psy.invokes.invoke_list[0].schedule
```

Comment:
Apologies again but I think we need this bit of the test back.

Reply:
This is now a separate test - done